zerver/openapi/zulip.yaml

```diff
@@ -29451,6 +29461,9 @@ components:
             A [group-setting value][setting-values] defining the set of users
             who have permission to create a new topic in this channel.
 
+            Note that for [private channels with protected history](/help/channel-permissions#private-channels),
+            everyone can start new topics.
+
```

Comment:
I think we want this to say:

> `Note that this must be the `role:everyone` system group for private channels with protected history.`

Since we are validating that, not ignoring this setting, right? (Keep the link, I just didn't want to type it out).